rdmd.d

```diff
@@ -839,6 +835,102 @@ import std.stdio, std.algorithm, std.array, std.ascii, std.base64,
     std.zlib;
 ";
 
+/**
+    Joins together the code provided via an `--eval`
+    or `--loop` flag, ensuring a trailing `;` is
+    added if not already provided by the user
+
+    Params:
+        eval = array of strings generated by the
+               `--eval` or `--loop` rdmd flags
+
+    Returns:
+        string of code to be evaluated, corresponding
+        to the inner code of either the program or
+        the loop
+*/
+string innerEvalCode(string[] eval)
+{
+    import std.string: join, stripRight;
```

Comment:
Nit: space before and after `:` for selective imports (DStyle for official code).